containers/teuthology-dev/Dockerfile

```diff
@@ -28,16 +27,13 @@ RUN \
 COPY . /teuthology
 RUN \
     ./bootstrap
-COPY docs/docker-compose/teuthology/containerized_node.yaml /teuthology
-COPY docs/docker-compose/teuthology/.teuthology.yaml /root
-COPY docs/docker-compose/teuthology/teuthology.sh /
-RUN mkdir -p /etc/ansible
-COPY docs/docker-compose/teuthology/ansible_inventory/hosts /etc/ansible/
```

Comment:
Can you remind me why you are removing ansible files again?

Reply:
We probably shouldn't have ever actually copied files from ceph-sepia-secrets.git into the container image; we should probably use a volume for this in `docker-compose.yml`. `ceph-devstack` doesn't currently have that feature, but it could be added.